app/src/main/res/layout/profile_chooser_add_view.xml

```diff
@@ -20,7 +21,8 @@
 
     <TextView
       android:id="@+id/add_profile_text"
-      android:layout_width="wrap_content"
+      android:layout_width="112dp"
```

Comment:
Why this change?

Reply:
As per mock and also it was looking better when we applied this limit.. As you suggest i can remove its.

Reply:
Changed to wrap_content